File: src/lib-node/flower.js
function flower(x)
{
 check(is_str,x)
 
 const n=process.stdout.columns
 
 let s1=repeat("*",n)
 let s2=repeat("*",2)
 
 s2=concat(s2," ")
 s2=concat(s2,x)
 s2=concat(s2," ")
 s2=concat(s2,s1)
 s2=slice_l(s2,n)

 log(s2)
}

File: src/lib-node/flower.cs
fn flower x
 check is_str x
 
 let n process.stdout.columns
 
 let s1 repeat "*" n
 let s2 repeat "*" 2 
 let s2 concat s2 " "
 let s2 concat s2 x
 let s2 concat s2 " "
 let s2 concat s2 s1
 let s2 slice_l s2 n

 log s2
end
